netplan/cli/utils.py

```diff
@@ -23,6 +23,8 @@
 import argparse
 import subprocess
 import netifaces
+import re
+import glob
```

Comment:
If we decide to ditch the possibly-invalid default value for nm_interfaces, we can remove glob from here.

Reply:
Fixed.